### PR TITLE
DDCE-2721-Migrate reactive mongo to hmrc-mongo

### PR DESCRIPTION
**Pull request by anu9**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -6,7 +6,7 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.6.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.15")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.18")
 
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
```
Comment (PaulC-TDS) [inline project/plugins.sbt:12]: Missing plugin for `dependencyUpdates` (the command fails at the moment when running `run_all_tests.sh`):
`addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.6.3")`

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -5,22 +5,19 @@ object AppDependencies {
...
     "org.scalatestplus"           %% "scalatestplus-scalacheck" % "3.1.0.0-RC2",
-    "org.scalatestplus"           %% "mockito-3-12"             % "3.2.10.0",
-    "org.pegdown"                 %  "pegdown"                  % "1.6.0",
-    "org.jsoup"                   %  "jsoup"                    % "1.14.3",
+    "org.mockito"                 %% "mockito-scala-scalatest"  % "1.17.12",
     "com.typesafe.play"           %% "play-test"                % PlayVersion.current,
-    "org.mockito"                 %  "mockito-all"              % "1.10.19",
     "org.scalacheck"              %% "scalacheck"               % "1.16.0",
     "com.github.tomakehurst"      % "wiremock-standalone"       % "2.27.2",
     "com.vladsch.flexmark"        % "flexmark-all"              % "0.62.0"
```
Comment (PaulC-TDS) [inline project/AppDependencies.scala:23]: "org.scalatest"               %% "scalatest"                % "3.2.12" can be upgraded to 3.2.14
"org.scalacheck"              %% "scalacheck"               % "1.16.0", can be upgraded to 1.16.0

File: run_all_test.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+sbt clean scalastyleAll compile coverage test coverageOff coverageReport dependencyUpdates
```
Comment (PaulC-TDS) [inline run_all_test.sh:3]: No newline at end of file

File: build.sbt [hunk 4]
```diff
@@ -72,3 +72,5 @@ lazy val testSettings: Seq[Def.Setting[_]] = Seq(
 )
 
 dependencyOverrides ++= AppDependencies.overrides
+
+addCommandAlias("scalastyleAll", "all scalastyle test:scalastyle it:scalastyle")
```
Comment (PaulC-TDS) [inline build.sbt:76]: Contains deprecated code, `in` can be replaced with `/`

Comment (anu9) [reply build.sbt:76]: done

**Review by PaulC-TDS — APPROVED** [review]

**Pull request merged**